modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridCachePartitionsStateValidationTest.java

```diff
@@ -323,4 +330,136 @@ public void waitUntilAllSingleMessagesAreSent() throws IgniteCheckedException {
             allSingleMessagesSent.get();
         }
     }
+
+    /**
+     *  Test that partitions state validation works correctly and show partition size always (GG-24471):
+     *  1. Only partition counters are inconsistent.
+     *
+     * @throws Exception If failed.
+     */
+
+    /** Start three-nodes grid: first node (coordinator) with LogListener and two nodes without*/
+    private IgniteEx startGridWithFirsLoggerNode() throws Exception {
```

Comment:
This method can still be extended. If you add 2 boolean agruments, you can move logic from all three tests here.